app/views/admin/schools/induction_coordinators/name_different.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :title, "Error: name different" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-header-xl">
+      The name does not match
+    </h1>
+
+      <p class="govuk-body">An induction coordinator with that email address is already registered as <%= @existing_name %>.</p>
+
+    <p>Either change the name to match, or contact the Register and Partner team</p>
```

Comment:
That copy feels wrong. Do we call ourselves `R&P` to the users? Should this be a support email link?

Reply:
We do to the frontline support staff. This is the admin interface remember

Reply:
Ah, right. Ehh, frustrating that we have many names, but nothing that we can easily do about it.